src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,115 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRXConfiguration;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.CANcoderConfigurator;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.Encoder;
+
+public class ArmConstants {
+    private static final int
+            ANGLE_MASTER_MOTOR_ID = 1,
+            ELEVATOR_MASTER_MOTOR_ID = 2,
+            ANGLE_FOLLOWER_MOTOR_ID = 3,
+            ELEVATOR_FOLLOWER_MOTOR_ID = 4,
+            ANGLE_ENCODER_ID = 5,
+            ELEVATOR_ENCODER_ID = 6;
+
+    private static final CANSparkMax
+            ANGLE_MASTER_MOTOR = new CANSparkMax(ANGLE_MASTER_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            ELEVATOR_MASTER_MOTOR = new CANSparkMax(ELEVATOR_MASTER_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            ANGLE_FOLLOWER_MOTOR = new CANSparkMax(ANGLE_FOLLOWER_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            ELEVATOR_FOLLOWER_MOTOR = new CANSparkMax(ELEVATOR_FOLLOWER_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    private static final TalonSRX ELEVATOR_ENCODER = new TalonSRX(ELEVATOR_ENCODER_ID);
+    private static final CANSparkMax.IdleMode
+            ELEVATOR_MASTER_IDLE_MODE = CANSparkMax.IdleMode.kCoast,
+            ANGLE_MASTER_IDLE_MODE = CANSparkMax.IdleMode.kCoast,
+            ELEVATOR_FOLLOWER_IDLE_MODE = CANSparkMax.IdleMode.kCoast,
+            ANGLE_FOLLOWER_IDLE_MODE = CANSparkMax.IdleMode.kCoast;
```

Comment:
The idle mode should be the same for a master motor and its follower, and should be brake for both by default.